doc/source/ray-air/examples/dolly_lightning_fsdp_finetuning.ipynb

```diff
@@ -89,7 +89,7 @@
     "O, pardon me, Signior Gremio; I would fain be doing.\n",
     "```\n",
     "\n",
-    "Here, we have adopted similar pre-processing logic from another demo: {ref}`GPT-J-6B Fine-Tuning with Ray AIR and DeepSpeed <gptj_deepspeed_finetune>`."
```

Comment:
Not sure about the scope of this PR but this doc needs to be updated as well? 
It's still using LightningTrainer?

Reply:
Yeah we have to update the fsdp example as well, but it's not a P0. Consider the timeline, I'll probably have to update it after the branch cut.